flink-rpc/flink-rpc-akka/src/main/resources/META-INF/services/org.apache.flink.runtime.rpc.RpcSystem

```diff
@@ -0,0 +1,16 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+org.apache.flink.runtime.rpc.akka.AkkaRpcSystem
```

Comment:
Out of curiosity: I guess that's a requirement by Akka?!

Reply:
no, this is required because we load it with a `ServiceLoader`

Reply:
Essentially, this file acts as a lookup for available implementations of an interface.

Reply:
Thanks for clarification.